src/AspNet.Security.OAuth.Discord/DiscordAuthenticationOptions.cs

```diff
@@ -28,6 +27,14 @@ public class DiscordAuthenticationOptions : OAuthOptions
         /// </summary>
         public string DiscordAvatarFormat { get; set; } = Urls.AvatarUrlFormat;
 
+        /// <summary>
+        /// Controls how the authorization flow handles existing authorizations.
+        /// The default value of this property is "Omit" and the "prompt" query string parameter will not be appended to the Authorization Endpoint URL.
+        /// Assigning this property any other value will automatically append the "prompt" query string parameter to the Authorization Endpoint URL,
+        /// with the corresponding value.
+        /// </summary>
+        public DiscordAuthenticationPrompt DiscordAuthenticationPrompt { get; set; } = DiscordAuthenticationPrompt.Omit;
```

Comment:
Would it be better to make this a `string`, rather than an enum? Then if Discord adds any new parameters in the future we don't need to do any additional development work to support it.

Reply:
Sure thing, I can make that change.

Reply:
Would you like to have the presently known available values available as Constants?

Reply:
> Would you like to have the presently known available values available as Constants?

It's a good idea 👍